charts/pvc-autoresizer/templates/_helpers.tpl

```diff
@@ -60,3 +60,44 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create the name of the webhook service
+*/}}
+{{- define "pvc-autoresizer.webhookService" -}}
+{{- printf "%s-controller" (include "pvc-autoresizer.fullname" .) -}}
+{{- end -}}
+
+{{/*
+Create the name of the webhook cert secret
+*/}}
+{{- define "pvc-autoresizer.webhookCertSecret" -}}
+{{- printf "%s-tls" (include "pvc-autoresizer.fullname" .) -}}
+{{- end -}}
+
+{{/*
+Generate certificates for webhook
+*/}}
+{{- define "pvc-autoresizer.webhookCerts" -}}
+{{- if not .Values.webhook.caBundle }}
+{{- if not .Values.webhook.existingCertManagerIssuer }}
+{{- if not (index .Values "cert-manager" "enabled") }}
+{{- $serviceName := (include "pvc-autoresizer.webhookService" .) -}}
+{{- $secretName := (include "pvc-autoresizer.webhookCertSecret" .) -}}
+{{- $secret := lookup "v1" "Secret" .Release.Namespace $secretName -}}
```

Comment:
I didn't think it was necessary to consider existing secrets. If there is a reason to consider it, would tell us? Is it a problem if the secret is re-created each time it is deployed?

Reply:
This was done precisely in order to avoid its constant re-creation. Increasing idempotency when deploying applications.
But if this bothers you, I can remove this part of the code.

Reply:
Thank you. I would like to be left with documentation of how to re-create the secret in case it expires. The document about cert-manager in `deploy/README.md`, would you add some notes in new section on the same file?

Reply:
Hello, unfortunately I can't find this readme catalogue.
In short, the certificate is issued for 10 years. After 10 years, it is enough to manually delete the old secret and update the application using `helm upgrade`. Then a new one will be created.

Reply:
Sorry. I made a mistake. Would you please add that note on values.yaml? I noticed that the description is empty, so I would like you to add a comment on values.yaml like other parameters and re-generate README.md.